test/bouture.test.js

```diff
@@ -267,3 +267,45 @@ describe(`Bouture.ul( ['a', 'b', 'c'].map(text => Bouture.li(text)))`, () => {
     expect(ul.children[0].textContent).to.equal('a')
   })
 })
+
+describe(`Bouture.button({onclick: e => { state = 37 }})`, () => {
+  it('should create a button and bind with an onclick event', () => {
+    var state = 0
+    const click = new MouseEvent('click')
+    const button = Bouture.button({onclick: e => { state = 37 }}).getElement()
+    button.dispatchEvent(click)
+
+    expect(state).to.equal(37)
+  })
+})
+
+describe(`Bouture.button({onclick: 'Yo'`, () => {
+  it('should create a button without onclick attribute', () => {
+    const button = Bouture.button({onclick: 'Yo'}).getElement()
+
+    expect(button.getAttribute('onclick')).to.equal(null)
+  })
+})
+
+describe(`Bouture.button({onair: e => console.log('On Air', e)})`, () => {
+  it('should not bind with a wrong event', () => {
+    const button = Bouture.button({onclick: e => console.log('click', e)}).getElement()
+
+    expect(typeof button.onair).to.equal('undefined')
+  })
+})
+
+describe(`Bouture.button({onceclick: 'Yo'`, () => {
+  it('should create a button and bind an onclick event who can only used once', () => {
```

Comment:
:+1: 

I think we need the same test but with `onClick` and making sure the listener is actually called twice when 2 events are dispatched